TopMovies/View/CategoryVC/MovieTableViewCell.swift

```diff
@@ -0,0 +1,182 @@
+//
+//  MovieTableViewCell.swift
+//  TopMovies
+//
+//  Created by Macbook Pro  on 17.02.2021.
+//
+
+import UIKit
+import SnapKit
+import Nuke
+
+// MARK: - Props struct -
+struct MovieTableViewCellProps {
+  let posterURL: URL?
+  let posterPlaceholderImage: UIImage
+  let titleLabelText: String
+  let releaseDateLabelText: String
+  let ratingAndVotesLabelText: String
+  let movieIsNew: Bool
+  let movieForAdult: Bool
+  let descriptionLabeltext: String
+}
+extension MovieTableViewCellProps {
+  init?(movie: Movie?) {
+    guard let movie = movie else { return nil }
+    posterURL = movie.poster
+    posterPlaceholderImage = UIImage(named: "moviePlaceholder") ?? UIImage()
+    titleLabelText = movie.title
+    releaseDateLabelText = "Release: \(MovieTableViewCellProps.prettyDate(movie.releaseDate))"
+    ratingAndVotesLabelText = "\(movie.rating) / 10 out of \(movie.voteCount) votes"
+    movieIsNew = MovieTableViewCellProps.isNewMovie(dateString: movie.releaseDate)
+    movieForAdult = movie.adult
+    descriptionLabeltext = movie.description
+  }
+  
+  static func prettyDate(_ dateString: String) -> String {
+    guard
+      let date = DateFormatter.cached(withFormat: "yyyy-mm-dd")
+        .date(from: dateString)
+    else { return "no date" }
+    return DateFormatter.cached(withFormat: "MMM, dd - yyyy")
+      .string(from: date)
+  }
+  static func isNewMovie(dateString: String) -> Bool {
+    return abs((DateFormatter.cached(withFormat: "yyyy-mm-dd")
+                  .date(from: dateString) ?? Date())
+                .months(from: Date())) < 3
```

Comment:
Don't forget to use Xcode re-indendation using `Ctrl + I`. For entire file - `Cmd + A -> Ctrl + I`.

This is something that should be done every time before you run your project just not to forget about that ;)